### PR TITLE
Add box-styles: border box to toggle :before

### PR DESCRIPTION
**Pull request by joecowton1**
## What is the purpose of this change?

When implemented in projects using a CSS reset, the positioning of the tick on the web toggle was broken.

## What does this change?

This PR adds the specific section of the CSS reset to the parent div of the toggle component, and then adds the fix for this to the `:before` element that was being broken.

-   add CSS reset styles specific to this component
-   updates `:before` styling `width` and `height` to fix toggle tick

## Screenshots

**Before (with CSS reset)**
Screenshot 2022-01-20 at 14 58 30

**After (with CSS reset)**
Screenshot 2022-01-20 at 14 58 40

### Accessibility

-   [x] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [x] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [x] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [x] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### Cross browser and device testing

-   [ ] Tested with touch screen device

### Responsiveness

-   [x] Tested at all breakpoints
-   [x] Tested with with long text
-   [x] Stretched to fill a wide container

### Documentation

-   [x] Full API surface area is documented in the README
-   [x] Examples in Storybook

### CODE REVIEW

**Comment by joecowton1** [comment]
@SiAdcock @sndrs little bump to review this tweak if you've time please?

File: packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts [hunk 1]
```diff
@@ -106,6 +112,7 @@ export const webStyles = css`
 		border-bottom: 2px solid ${success[400]};
 		border-right: 2px solid ${success[400]};
 		transition: opacity 0.1s ease-in;
+		box-sizing: content-box;
```
Comment (SiAdcock) [inline packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts:115]: If we tweak the width and height of the the `:before`, we can achieve a similar effect without the need to override the `box-sizing`:

```css
height: 0.7rem;
width: 0.4rem;
```
**Chrome**
Screenshot 2022-01-24 at 10 01 58

**Safari**
Screenshot 2022-01-24 at 10 02 24


**Firefox**
Screenshot 2022-01-24 at 10 02 49


What do you think?

**Review by SiAdcock — APPROVED** [review]
I've added a small suggestion, but it's not a blocker 🙂 This looks great

File: packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts (hunk 1 above)
Comment (joecowton1) [reply packages/@guardian/source-react-components-development-kitchen/src/toggle-switch/styles.ts]: Thanks, yeah that's much better! Have implemented and pushed changes here.

**Review by SiAdcock — APPROVED** [review]

**Review by sndrs — APPROVED** [review]
nice oen!

**Comment by sndrs** [comment]
released in @guardian/source-react-components-development-kitchen@0.0.29

**Pull request merged**